reports.go

```diff
@@ -46,44 +66,119 @@ type Report struct {
 	StateCriminalSearchIDs           []string   `json:"state_criminal_search_ids,omitempty"`
 	DocumentIDs                      []string   `json:"document_ids,omitempty"`
 	GeoIDs                           []string   `json:"geo_ids,omitempty"`
+	IdentityDocumentVerificationID   string     `json:"identity_document_verification_id"`
 }
 
 // Unmarshal ...
 func (r *Report) Unmarshal(b []byte) error {
 	return json.Unmarshal(b, &r)
 }
 
-const createReportPath = "/v1/reports"
+const ssnTracePath = "/v1/ssn_traces"
 
-func (c *Client) CreateReport(reqPayload *CreateReportRequest) (*Report, error) {
-	rel, err := url.Parse(createReportPath)
+// GetSSNTrace return the ssn trace screening report
+func (r *Report) GetSSNTrace(ssnTraceID string, c *Client) (*SSNTrace, Error) {
+	// since ID is empty we don't have anything to return. the caller should make sure if ssnTraceID is nil
+	if ssnTraceID == "" {
+		return nil, NewError([]int{
+			http.StatusOK,
+		}, nil)
+	}
+
+	rel, err := url.Parse(ssnTracePath)
 	if err != nil {
-		return nil, err
+		return nil, NewError([]int{
+			http.StatusOK,
+		}, nil)
 	}
 
 	u := *c.BaseURL
-	u.Path = path.Join(u.Path, rel.String())
+	u.Path = path.Join(u.Path, rel.String(), ssnTraceID)
 
-	b, err := json.Marshal(reqPayload)
+	req, err := http.NewRequest(http.MethodGet, u.String(), nil)
 	if err != nil {
-		return nil, err
+		return nil, NewError([]int{
+			http.StatusOK,
+		}, nil)
 	}
+	req.Header.Add("Content-Type", "application/json")
+	req.SetBasicAuth(c.APIKey, "")
 
-	req, err := http.NewRequest(http.MethodPost, u.String(), bytes.NewBuffer(b))
+	resp, err := c.client.Do(req)
 	if err != nil {
-		return nil, err
+		return nil, NewError([]int{
+			http.StatusOK,
+		}, resp)
 	}
-	req.Header.Add("Conent-Type", "application/json")
+
+	if resp.StatusCode != http.StatusOK {
+		return nil, NewError([]int{
```

Comment:
@chandradeepak I think it make it more obvious what the error is, let's just rename it to `ResponseError`.

Reply:
do you mean rename NewError to ResponseError right. i did that